--- src/caffe/CMakeLists.txt	2020-09-15 19:06:01.512521268 -0400
+++ src/caffe/CMakeLists.txt.new	2020-09-15 20:20:59.333469375 -0400
@@ -44,6 +44,7 @@
 #target_include_directories(caffeproto PUBLIC ${PROTOBUF_INCLUDE_DIR})
 
 list(INSERT Caffe_LINKER_LIBS 0 PUBLIC caffeproto) # note, crucial to prepend!
+list(INSERT Caffe_LINKER_LIBS 0 PUBLIC ${CONAN_LIBS} ${vecLib_LINKER_LIBS})
 
 # --[ Caffe library
 
